### PR TITLE
Make 2.0 codegen compatible with 1.0 projects until multi-targeting i…

### PR DESCRIPTION
**Pull request by NTaylorMullen**
…s complete.

#1637 

Working on verifying this inside of Visual Studio.

### CODE REVIEW

File: src/Microsoft.AspNetCore.Razor.Language/Extensions/SectionTargetExtension.cs [hunk 1]
```diff
@@ -7,6 +7,8 @@ namespace Microsoft.AspNetCore.Razor.Language.Extensions
 {
     public sealed class SectionTargetExtension : ISectionTargetExtension
     {
+        private const string DefaultWriterName = "__razor_section_writer";
```
Comment (ajaybhargavb) [inline src/Microsoft.AspNetCore.Razor.Language/Extensions/SectionTargetExtension.cs]: Leave a comment here as well.

Comment (ajaybhargavb) [inline src/Microsoft.AspNetCore.Razor.Language/Extensions/SectionTargetExtension.cs]: Leave a comment here as well.

**Comment by NTaylorMullen** [comment]
Working on verifying this within a clean VSUWPT install (did locally on my box), merged to get Coherence-Signed going.

**Comment by NTaylorMullen** [comment]
Verified in the current VSUWPT channel by dropping updated binaries in.

**Pull request merged**